index.html

```diff
@@ -149,11 +149,15 @@ <h3>
         </h3>
         <p>
           The [=note_taking=] <code>new_note_url</code> member is a <a>URL</a>
-          [=manifest/within scope=] of the manifest, to open when the user
-          wants to take a new note using the web application (e.g., from an
-          operating system shortcut icon or keyboard shortcut). Defining this
-          member indicates that the web application is a note-taking
-          application.
+          to open when the user wants to take a new note using the web
+          application (e.g., from an operating system shortcut icon or keyboard
+          shortcut). Defining this member indicates that the web application is
```

Comment:
I know this isn't the point of this PR, but I figure since we're making this text more formal, you could probably do it here at the same time.

This is pretty non-specific, especially since it doesn't define what a "note-taking application" is. I think the way to fix it is not to define "note-taking application", but to remove this sentence and replace it with an algorithm for launching the note URL. Similar to how Web Share Target defines [in an algorithm how to launch the share target URL](https://w3c.github.io/web-share-target/#launching-the-web-share-target), and file handling should do this too if it doesn't already.

Some roughly thought out potential spec text here:

"When the user agent needs to invoke the note-taking capabilities of an application with `WebAppManifest` _manifest_, run the following steps:

1. Let _browsing context_ be the result of creating a [new top-level browsing context](https://html.spec.whatwg.org/multipage/#top-level-browsing-context).
2. [Navigate](https://html.spec.whatwg.org/multipage/#navigate) _browsing context_ to a new [Request](https://fetch.spec.whatwg.org/#concept-request) whose method is `"GET"` and url is _manifest_`[new_note_url]`."

(I copied this from the last two steps of Web Share Target algorithm which should help get the Respec syntax right.